commands/user.go

```diff
@@ -84,6 +84,16 @@ If MFA enforcement is enabled, the user will be forced to re-enable MFA as soon
 	RunE:    withClient(resetUserMfaCmdF),
 }
 
+var DeleteAllUsersCmd = &cobra.Command{
+	Use:     "deleteall",
```

Comment:
Should we use delete-all instead?

Reply:
I'm really 0/5 on this, just used `deleteall` for consistency with the legacy CLI

Reply:
Oh, I see 👍